guides/common/modules/ref_predefined-roles.adoc

```diff
@@ -7,48 +7,145 @@ To view the exact set of permissions a predefined role grants, display the role
-| Ansible Tower Inventory Reader | View facts, hosts, and host groups.
-| Bookmarks manager | Create, edit, and delete bookmarks.
-| Boot disk access | Download the boot disk.
-| Compliance manager | View, create, edit, and destroy SCAP content files, compliance policies, and tailoring files.
+|
+
+|Ansible Tower Inventory Reader
+|View facts, hosts, and host groups.
+|
+
+|Bookmarks manager
+|Create, edit, and delete bookmarks.
+|
+
+|Boot disk access
+|Download the boot disk.
+|
+
+|Compliance manager
+|View, create, edit, and destroy SCAP content files, compliance policies, and tailoring files.
+
 View compliance reports.
-| Compliance viewer | View compliance reports.
-| Create ARF report | Create compliance reports.
-| Default role | The set of permissions that every user is granted, irrespective of any other roles.
-| Discovery Manager| View, provision, edit, and destroy discovered hosts and manage discovery rules.
-| Discovery Reader | View hosts and discovery rules.
-| Edit hosts | View, create, edit, destroy, and build hosts.
-| Edit partition tables | View, create, edit and destroy partition tables.
-| Manager | View and edit global settings.
-| Organization admin | An administrator role defined per organization.
+|
+
+|Compliance viewer
+|View compliance reports.
+|
+
+|Create ARF report
+|Create compliance reports.
+|
+
+|Default role
+|
+ifndef::foreman-deb,foreman-el[]
+View tasks and jobs invocations.
+endif::[]
+
+ifdef::foreman-deb,foreman-el[]
+The permissions this role provides depend on whether the remote execution and tasks plug-ins are enabled.
+endif::[]
+|{Project} automatically assigns this role to every user in the system.
+
+|Discovery Manager
+|View, provision, edit, and destroy discovered hosts.
+
+Manage discovery rules.
+|
+
+|Discovery Reader
+|View hosts and discovery rules.
+|
+
+|Edit hosts
+|View, create, edit, destroy, and build hosts.
+|
+
+|Edit partition tables
+|View, create, edit, and destroy partition tables.
+|
+
+|Manager
+|View and edit global settings.
+|
+
+|Organization admin
+|All permissions except permissions for managing organizations.
+|An administrator role defined per organization.
 The role has no visibility into resources in other organizations.
-| Red{nbsp}Hat Access Logs | View the log viewer and the logs.
-| Remote Execution Manager | Control which roles have permission to run infrastructure jobs.
-| Remote Execution User | Run remote execution jobs against hosts.
-| Site manager | A restrained version of the Manager role.
-| System admin a|
-* Edit global settings in *Administer* > *Settings*.
-* View, create, edit and destroy users, user groups, and roles.
-* View, create, edit, destroy, and assign organizations and locations but not view resources within them.
-
-Users with this role can create users and assign all roles to them.
+
+By cloning this role and assigning an organization, you can delegate administration of that organization to a user.
+
+|Red{nbsp}Hat Access Logs
```

Comment:
I did not find this role in Foreman 3.7. Does it depend on RH Insights plug-in?

Reply:
It looks like the role is called Auditor now. At least judging from what I can see in the web UI. I'll update the list.